### ISSUE TITLE
Feature: Taskbar Icon - right click - immediate sync option

### ISSUE THREAD

**Issue by frederikb96** [owncloud/client#9569]
I guess it would be very beneficial to have the option to restart sync (force sync now) for all folders in the context menu when right-clicking the owncloud icon in the task bar. The option could be directly displayed below the “Pause synchronization” option.

Actually, I guess the “force synchronization” option is much more often used than the “Pause synchronization” option, as it is used to trigger an update immediately if you have just changed a specific file that shall be synced to another device for further usage. Currently, one has to open the client, look for the folder containing this file and force the sync for that specific folder.

Moreover, it is also often the case, that some file (like a notebook, which is saved frequently) is changed during sync, which leads to the case, that this file is not automatically synced anymore, until owncloud is restarted, or the user forces a new sync. In this case, it is also a benefit to be able to simply right-click the owncloud icon to force a new sync.

**Comment by ChrisEdS** [comment]
I like that idea! Sounds very handy.

+1 from my side.

**Comment by TheOneRing** [comment]
Actually the force sync now should not be used too often as it will massively increase the power usage.
A force sync will perform a full local discovery while a normal sync is incremental.
Is there any reason you need to force a sync rather often?

**Comment by ChrisEdS** [comment]
I think that force sync is meant here more like "sync immediately".

**Comment by TheOneRing** [comment]
For changes on the server, we should detect them every 30s-1min.
Local changes should be detected immediately (on Windows once unlocked).
So why do we need to force the sync.

**Comment by frederikb96** [comment]
I appreciate that it is very important to not add unnecessary features and keep the client minimal. However, this option seems to be quite an improvement.

I did not consider the difference between normal sync and force sync. The normal sync option would already be beneficial to allow an immediate sync for linux users like me when I have changed a local file like a notebook. 

It would also be benefical for all users who work with two devices in parallel since in this case, it would be necessary to trigger a sync on the device which is used to download the new file version. Windows user would also have too wait for 30 - 60 seconds otherwise.

If this feature is not going to be implemented, is there actually a way to trigger a sync from the command line? This would also solve the problem.

About the force sync option: The only use case I have for this is when I enable auto save for files I am working on. This always leads to the error message that a file is changed during sync. And only a force sync, a owncloud client restart or waiting a very long time, clears that state for the file for me, such that the sync is continued for that specific file.

However, I see that having a force sync option as a right click is not a proper solutions and I will rather check if I will open another issue for that.

**TLTR**
To sum things up: A right l click option to trigger a normal immediate sync via the owncloud icon on the task bar, is the feature I am requesting. It would improve the possibility to work with two devices in parallel, which are used to work on the same files.

**Comment by TheOneRing** [comment]
You can retrigger sync for specific files (single file force sync) in the issue table, using the right click context menu.
